.solhint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "solhint:recommended",
+  "plugins": [],
+  "rules": {
+    "avoid-suicide": "error",
+    "avoid-sha3": "warn",
+    "compiler-version": ["error","^0.8.7"],
+    "no-empty-blocks": "off",
+    "func-visibility": ["warn",{"ignoreConstructors":true}]
+  }
+  
+}
```

Comment:
change all warn to error,
my recommended list of rules:
    "no-unused-vars": "error",
    "const-name-snakecase": "error",
    "contract-name-camelcase": "error",
    "event-name-camelcase": "error",
    "func-name-mixedcase": "error",
    "func-param-name-mixedcase": "error",
    "modifier-name-mixedcase": "error",
    "private-vars-leading-underscore": "error",
    "var-name-mixedcase": "error",
    "imports-on-top": "error",
    "function-max-lines": ["error", 50],
    "max-line-length": ["error", 120],
    "quotes": ["error","double"],
    "ordering": "error",
    "check-send-result": "error",
    "state-visibility": "error"